package.json

```diff
@@ -18,33 +18,30 @@
     "test:unit:coverage": "sfdx-lwc-jest --coverage -- --runInBand",
     "prettier": "prettier --write \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
     "prettier:verify": "prettier --list-different \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
-    "build:playground": "rollup --config ./rollup/rollup.config.js"
+    "build:playground": "rollup --config ./rollup/rollup.config.js",
+    "pre-commit": "lint-staged && npm run lint",
+    "pre-push": "npm run test:coverage",
+    "prepare": "husky install"
```

Comment:
```suggestion
    "pre-push": "npm run test:coverage"
```

Reply:
What is bothering you here @victorgz ?

Reply:
@scolladon arg forget about it !!! it was just a lapsus, I read by mistake in my head: "prepare" : "echo husky install", and I thought it was something you had used for testing 😃